Makefile

```diff
@@ -202,8 +202,12 @@ kafka:
 ifeq ($(OLM),true)
 	@echo Skipping kafka-operator deployment, assuming it has been installed via OperatorHub
 else
+	@kubectl create clusterrolebinding strimzi-cluster-operator-namespaced --clusterrole=strimzi-cluster-operator-namespaced --serviceaccount ${KAFKA_NAMESPACE}:strimzi-cluster-operator 2>&1 | grep -v "already exists" || true
```

Comment:
https://strimzi.io/docs/master/#deploying-cluster-operator-to-watch-whole-cluster-str